<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BurstView</class>
 <widget class="QWidget" name="BurstView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>533</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTableWidget" name="tableBursts">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;double click to plot the burst&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="JKQtPlotter" name="widgetPlotCorr" native="true"/>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,1">
       <item row="0" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Binwidth</string>
         </property>
         <property name="buddy">
          <cstring>doubleSpinBoxBinwidth</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Transparency for Burst</string>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxOffset">
         <property name="suffix">
          <string>ms</string>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>1.000000000000000</double>
         </property>
         <property name="value">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="11">
        <widget class="JKQtPlotter" name="widPlotBurst" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QSpinBox" name="spinBoxAlpha">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>40</number>
         </property>
        </widget>
       </item>
       <item row="0" column="10">
        <widget class="QPushButton" name="pushButtonSaveBurst">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="0" column="9">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>290</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Transparency for excitation periods</string>
         </property>
         <property name="buddy">
          <cstring>spinBoxAlpha</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Offset</string>
         </property>
         <property name="buddy">
          <cstring>doubleSpinBoxOffset</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QComboBox" name="comboBoxPlotType">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>Intensity</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TTTR</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxBinwidth">
         <property name="suffix">
          <string>us</string>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="value">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QSpinBox" name="spinBoxAlphaBurst">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>60</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>JKQtPlotter</class>
   <extends>QWidget</extends>
   <header>jkqtplotter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
